### PR TITLE
New utility scripts

### PR DESCRIPTION
**Pull request by anara123**

### CODE REVIEW

File: Scripts/script-UnzipFile.yml [hunk 1]
```diff
@@ -0,0 +1,83 @@
...
+  with zipfile.ZipFile(filePath,"r") as zip_ref:
+      zip_ref.extractall(pwd=password)
+      for name in zip_ref.namelist():
+          if name == os.path.basename(name):
+              filenames.append(name)
+
+  results = []
+  for filename in filenames:
+      uniqueName = demisto.uniqueFile()
+      subprocess.call(['mv', filename, demisto.investigation()['id'] + '_' + uniqueName])
```
Comment (aviadl) [inline Scripts/script-UnzipFile.yml]: Not sure that this is the correct way moving files in the war room.
You should write to new file using our utilities.

File: Scripts/script-ParseCSV.yml [hunk 2]
```diff
@@ -36,8 +36,9 @@ script: |
   parseip = int(dArgs['ips']) if 'ips' in dArgs else None
   parsedomain = int(dArgs['domains']) if 'domains' in dArgs else None
   parsehash = int(dArgs['hashes']) if 'hashes' in dArgs else None
+  parseAll = demisto.args()['parseAll'] if 'parseAll' in dArgs else None
 
-  if not parseip and not parsedomain and not parsehash:
+  if not parseip and not parsedomain and not parsehash and not parseAll:
```
Comment (aviadl) [inline Scripts/script-ParseCSV.yml]: Remove the word `Please`

File: Scripts/script-ExportToCSV.yml [hunk 3]
```diff
@@ -0,0 +1,73 @@
...
+
+  var csvString = convertToCSV(args.csvArray);
+  var createdFileID = saveFile(csvString);
+
+  return {
+      Type: 9,
+      FileID: createdFileID,
+      File: args.fileName,
+      Contents: args.fileName,
+      HumanReadable: args.fileName
```
Comment (aviadl) [inline Scripts/script-ExportToCSV.yml]: file type should probably be 3
No need for HumanReadable

**Comment by dorsha** [comment]
@anara123 - can we merge this one?

**Comment by anara123** [comment]
@dorsha I think we can merge this. And open new issue to fix the @aviadl review

**Pull request merged**